lnwallet/wallet.go

```diff
@@ -1512,6 +1511,36 @@ func (l *LightningWallet) handleSingleFunderSigs(req *addSingleFunderSigsMsg) {
 	chanState.FundingOutpoint = *req.fundingOutpoint
 	fundingTxIn := wire.NewTxIn(req.fundingOutpoint, nil, nil)
 
+	// We received funding_created, update the ChannelContextSigner.
+	// FIXME - This should probably be moved to where open_channel
```

Comment:
the funding outpoint won't be available in `open_channel`.  also, in https://github.com/rust-bitcoin/rust-lightning/pull/723/files#diff-79ec46664bd0cf85b6be0ddb567527a7R1519 I move the call to the `funding_created` protocol step in the LDK integration

Reply:
Removed FIXME, will update `lightning-signer:docs` sequence diagram to match.